app/Http/Kernel.php

```diff
@@ -60,7 +62,7 @@ class Kernel extends HttpKernel
         'can' => \Illuminate\Auth\Middleware\Authorize::class,
         'guest' => \App\Http\Middleware\RedirectIfAuthenticated::class,
```

Comment:
@taylorotwell you added this middleware in https://github.com/laravel/framework/pull/47040/files#diff-6a190f6e68c89dafd0a08692e63e48982a2bf570884ef58f525c2637ea516ae1R10 but forgot to update it here.

```suggestion
        'guest' => \Illuminate\Auth\Middleware\RedirectIfAuthenticated::class,
```

Reply:
Probably also delete the `App\Http\Middleware\RedirectIfAuthenticated` middleware class after this.